colaraz/cms/templates/widgets/header.html

```diff
@@ -302,12 +305,14 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
                     <span class="icon-bar bottom-bar"></span>
                 </button>
                 <ul>
-                    <li><a href="#"><span class="ecosystem-link"></span>Ecosystem</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('RECRUITMENT_URL'))}"><span class="recruiting-link"></span>Recruiting</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('MANAGEMENT_URL'))}"><span class="projects-link"></span>Management</a></li>
-                    <li><a href="#"><span class="talent-link"></span>Performance</a></li>
-                    <li class="active"><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('DEVELOPMENT_URL'))}"><span class="development-link"></span>Development</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('SETTINGS_URL'))}"><span class="settings-link"></span>Settings</a></li>
+                  <li><a href="//${'{}.{}/cvsocial/activity?{}'.format(site_identifier, settings.COLARAZ_SUB_DOMAIN, role_based_urls.get('ecosystem'))}"><span class="ecosystem-link"></span>Ecosystem</a></li>
```

Comment:
Here on this link, you are getting `role_based_urls.get('ecosystem')` but here if we see your other PR: https://github.com/colaraz/edx-platform/pull/43/files#diff-a9adc29daa67aa294bae7142e2214082R446

Your function is returning the empty `Dict {}` while storing the URLs for the user. So, there is a chance that the user might not have the URLs. It will break here while getting them? 

Is it expected or you have just forgotten to add some checks here?

Reply:
If the api doesn't responses with 500 only then an empty dict will be stored. In that case our environment will not break but the links will be broken obviously. Can you please guide what should be the desired behaviour at this point?

Reply:
That's what I am asking do you want your system to break here? In case of URLs not found?

Reply:
Links will be broken incase the api doesn't give appropriate response, but when the user login's again those links will again get updated. so the links are totally dependent on colaraz's end. If we want to hide navbar for users then it will create inconsistency in UI which i don't think is a good option for now.

Reply:
Well, I am still not clear about the requirements here. But as per our discussion on call, it seems like you are confident that it will work fine.